### DISCUSSION TITLE
Format report level measures

### DISCUSSION THREAD

**Discussion by bluehenve** [sql-bi/Bravo#463] [Q&A] [answered]
Hi. 
In reports with SSAS as source, can report level measures also be formatted?
I've tried, but cannot see the report level measures in Bravo.
Regards, Henrik

**Comment by albertospelta** [comment, marked as answer]
Report-level measures cannot be formatted, this is an unsupported feature in current version. This is because the report-level measures are stored locally in the PBIX report and not in the tabular model (SSAS model / PBI dataset).

**Comment by bluehenve** [comment]
Ok 👍
